src/libraries/System.Runtime.Intrinsics/tests/X86/Sse2Tests.ConvertToNInt.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+namespace System.Runtime.Intrinsics.Tests.X86;
+
+public sealed partial class Sse2Tests
+{
+    //
+    // Sse2.ConvertToNInt is currently failing
+    // -> InvalidProgramException
+    // ¯\_(ツ)_/¯
+    //
```

Comment:
@tannergooding `Sse2.ConvertToNInt` throws an `InvalidProgramException`.

I think the problem is here (but I don't know):
https://github.com/dotnet/runtime/blob/15a8c1ab396e7476b6c8b8bc9c2afb3d9e1ee34e/src/coreclr/jit/hwintrinsiclistxarch.h#L386

Do you have a approach to fix?

Reply:
I'll have to pull down the code and debug most likely.

Reply:
Okay. What is your schedule like right now? Should I continue working on the tests for AVX, ...? or do you want to take a look at the current state before I do more commits?

Reply:
As described via Discord, this is now solved. Thanks for your help. 
The problem was a missing addition in lowerxarch.h